testFunction/saveGame.js

```diff
@@ -0,0 +1,67 @@
+const crawlGameSchedule = require("../utils/crawlGameSchedule");
+const crawlPlayerEntry = require("../utils/crawlPlayerEntry");
+const crawlPlayersInfo = require("../utils/crawlPlayersInfo");
+const Game = require("../models/Game");
+const Player = require("../models/Player");
+
+module.exports = async (dateNumber) => {
+  let dateString = dateNumber;
+  if (typeof date !== "string") {
```

Comment:
```suggestion
  if (typeof dateString !== "string") {
```
뭔가 이렇게 되어야 하는 거 같은데.. date는 어디에서 나온 건가요 👀